### ISSUE TITLE
Agenda only displays the matchs that are after the current date

### ISSUE THREAD

**Issue by clapoujade** [eric-taix/volley34-mobile#165]
Agenda should also display the matchs when their result are not yet entered/validated.

**Comment by eric-taix** [comment]
Not related to #15 which aims to display how many matches are planned in the following week (to be able to know if there's at least one free field to practice.

I'm not able to know what you'd like to do with this new feature? The agenda aims to display future events (matches, tournaments, meetings, ...). Yes a match can be played but not yet validated and I think that it doesn't have to be displayed in your agenda as the date is outdated. This issue will be closed except if I missed something.

But if you want to display matches which are postponed, then this is another feature not related to this issue (we should open another issue but for release v2).

**Comment by clapoujade** [comment]
It is not a feature...from m'y understanding, it looks like a bug.
I was looking for some matchs that were 'played' as the match date was over....but most propably the related results were not yet declared in volley34. 
During 5 days, these matchs were not visible at all. Not in the results, nor in the agenda. A kind of disturbing

**Comment by eric-taix** [comment]
As mentioned in my previous comment an agenda is mostly to see future events (even if in most agendas you can always see past events but in volley34, the agenda is only for future events) and this is the main feature of an agenda.

Yes there's a delay between the time a match is played and the time when the result is available (sometimes several days) but remember this is a human work to validate and edit the result ;-) This is how volley34 works not more, not less.

I understand that it could be disturbing but nothing we could do for now. In the future with new features in the mobile app (automatically send the result, signing capabilities, ...) results could be automatically validated? Who knows

**Comment by clapoujade** [comment]
Gain, I am not talking about the validation process.
Volley34 is working differently. Indeed, the full agenda is displayed. My request is more to display the full agenda but without the validated matchs. Perhaps there is a tecnhical issue that prevent this?

**Comment by eric-taix** [comment]
Yes you're right: volley34 seems to display the agenda for the current month (?) but it was not to solve postponed matches nor validated matches (I think even if I'm not sure).

Also it has been developed for a desktop with a big height. In mobile app the height is much more smaller and even if it's can scroll, I think that agenda must focus on what will happen in the next future to avis scrolling as much as possible.

But I understand your purpose: what about played matches not validated or postponed matches and how can we inform the user about these states and where. That's why I think that's it's a new feature which is out of scope of the current agenda. Perhaps not validated matches can be displayed in the result tab and postponed matches can be displayed in the agenda when the new date has been decided. But what about postponed matches without new date (not yet planned)?

As you can see there are a lot of business questions and I'm not talking about technical questions because even if there's a attribut for postponed matches, for now this attribut does not seems to be not filled.

While being answering you, I think we have to create 2 different issues (not validated, postponed) and close this one (but keep a reference on it). For sure this won't be fixed for the V1.
